### DISCUSSION TITLE
The domain <domain> is already owned by another Vercel account

### DISCUSSION THREAD

**Discussion by TTitcombe** [vercel/community#280] [Help] [answered]
I had added a domain I owned to a project on a different Vercel user's account so he could manage the site for me. Since then, I have decided to take full control of the site and went to add the domain. I have removed all host records relating to Vercel from my domain and the individual claims he removed the domain from his site (I cannot verify this - he is difficult to contact). When I try to add the domain to my project I get "Error: The domain <domain> is already owned by another Vercel account". How can I fix this?

**Comment by samcx** [comment, marked as answer]
Please request a case at https://vercel.com/support/request so we can take it from there. Make sure to reply with your case # here so we can take a look at it!
